app/controllers/admin/sales/discounted-events.js

```diff
@@ -1,4 +1,89 @@
 import Controller from '@ember/controller';
 import AdminSalesMixin from 'open-event-frontend/mixins/admin-sales';
+import { computed } from '@ember/object';
 
-export default Controller.extend(AdminSalesMixin);
+export default Controller.extend(AdminSalesMixin, {
+  columnNames: {
+    rowspan: [{
+      colname : 'Events',
+      class   : '',
+      span    : 2
+    }, {
+      colname : 'Discount Code',
+      class   : '',
+      span    : 2
+    }, {
+      colname : 'Marketer Email',
+      class   : '',
+      span    : 2
+    }],
+    colspan: [{
+      colname : 'Completed Orders',
+      class   : 'ui green inverted segment center aligned',
+      span    : 2
+    }, {
+      colname : 'Placed Orders',
+      class   : 'ui blue inverted segment center aligned',
+      span    : 2
+    }, {
+      colname : 'Pending Orders',
+      class   : 'ui yellow inverted segment center aligned',
+      span    : 2
+    }]
+  },
+  subColumnNames : ['Tickets', 'Sales', 'Tickets', 'Sales', 'Tickets', 'Sales'],
+  columnValues   : [
+    {
+      propertyName: 'eventName'
+    }, {
+      propertyName: 'code'
+    }, {
+      propertyName: 'email'
+    }, {
+      propertyName : 'sales.completed.sales_total',
+      type         : '$',
+      class        : 'right aligned'
+    }, {
+      propertyName : 'sales.placed.ticket_count',
+      type         : '',
+      class        : 'right aligned'
+    }, {
+      propertyName : 'sales.placed.sales_total',
+      type         : '$',
+      class        : 'right aligned'
+    }, {
+      propertyName : 'sales.pending.ticket_count',
+      type         : '',
+      class        : 'right aligned'
+    }, {
+      propertyName : 'sales.pending.sales_total',
+      type         : '$',
+      class        : 'right aligned'
+    }
+  ],
+
+  totalRow: computed(function() {
+    return [
+      {
+        value: this.get('totalCompletedTickets')
+      },
+      {
+        value: `US$ ${this.get('totalCompletedSales')}`
```

Comment:
Don't hardcode currency.

Reply:
It was hardcoded before as well https://github.com/fossasia/open-event-frontend/blob/00c680fcfd094427993ec431945d1560735fe37e/app/templates/admin/sales/discounted-events.hbs#L94

Reply:
Yeah, I know, but it's wrong. And since you are converting this into an ember table, this is the right time to fix it.

Reply:
so from do i get which currency is used?

Reply:
@CosmicCoder96 ?

Reply:
@YogeshSharma0201 Each event has a currency stored inside paymentCurrency attribute. See the event model.

Reply:
@CosmicCoder96 okay so if all the entries in the table can have different currency symbols then how would i calculate `total` of all the sales which is displayed at the bottom of the table

Reply:
I think the server should convert all the different currency values to one single currency which would also be easier to understand and analyze for someone viewing admin sales.

Reply:
@YogeshSharma0201 we have already commented out `total` due to this reason. You can skip the total part and move forward with this PR